pkg/core/object/fmt.go

```diff
@@ -343,10 +330,10 @@ func (v *FormatValidator) ValidateContent(o *object.Object) (ContentMeta, error)
 
 var errExpired = errors.New("object has expired")
 
-func (v *FormatValidator) checkExpiration(obj object.Object) error {
+func (v *FormatValidator) checkExpiration(obj object.Object, expirationRequired bool) error {
```

Comment:
add more context to the err? i think error in logs now would be `"object did not pass expiration check: missing expiration epoch attribute"` but it is now clear why it is bad